src/uucore/src/lib/features/checksum.rs

```diff
@@ -380,30 +493,16 @@ where
 {
     // if cksum has several input files, it will print the result for each file
     for filename_input in files {
-        let mut bad_format = 0;
-        let mut failed_cksum = 0;
-        let mut failed_open_file = 0;
         let mut correct_format = 0;
         let mut properly_formatted = false;
+        let mut res = ChecksumResult {
+            bad_format: 0,
+            failed_cksum: 0,
+            failed_open_file: 0,
+        };
```

Comment:
You can simplify this to `let mut res = ChecksumResult::default();` if you add `#[derive(Default)]` to `ChecksumResult`.